pkg/cloud/aws/services/cloudformation/bootstrap.go

```diff
@@ -281,6 +285,45 @@ func cloudProviderNodeAwsPolicy() *iam.PolicyDocument {
 	}
 }
 
+func makePath(dir string, fileName string) string {
+	if strings.HasSuffix(dir, "/") {
+		return fmt.Sprintf("%s%s.json", dir, fileName)
+	}
+	return fmt.Sprintf("%s/%s.json", dir, fileName)
+}
+
+func jsonForResource(resourceName string, resource interface{}) ([]byte, error) {
+
+	jsonToProcess, err := json.MarshalIndent(resource, "", "  ")
+	if err != nil {
+		return nil, fmt.Errorf("Error: failed to export resource %q: %v", resourceName, err)
+	}
+
+	jsonBytes, err := intrinsics.ProcessJSON(jsonToProcess, nil)
+	if err != nil {
+		return nil, fmt.Errorf("Error: failed to export resource %q: %v", resourceName, err)
+	}
+
+	return jsonBytes, err
+}
+
+// ExportBootstrapTemplate exports bootstrap AWS resources to JSON
+func (s *Service) ExportBootstrapTemplate(resourceDir string, accountID string) error {
+
+	template := BootstrapTemplate(accountID)
+	for rn, rv := range template.Resources {
```

Comment:
Any reason not to write a single template out using `template.JSON()`?

Reply:
It seemed more tractable having a JSON file per resource. That said, I don't have a strong preference one way or the other.